packages/text-input/src/TextInput/TextInput.spec.tsx

```diff
@@ -105,95 +102,8 @@ describe('packages/text-input', () => {
     spy.mockClear();
   });
 
-  describe('when the "state" is "valid"', () => {
-    test('displays checkmark icon when input is valid', () => {
-      const { getInputValue, isValid, isOptional } = renderTextInput({
-        value: validEmail,
-        state: State.Valid,
-        optional: true,
-        ...defaultProps,
-      });
-
-      expect(getInputValue()).toBe(validEmail);
-      expect(isValid()).toBe(true);
-      expect(isOptional()).toBe(false);
-    });
-
-    test('displays checkmark icon when input is valid even when input is disabled', () => {
-      const { getInputValue, isOptional } = renderTextInput({
-        value: validEmail,
-        state: State.Valid,
-        disabled: true,
-        ...defaultProps,
-      });
-
-      expect(getInputValue()).toBe(validEmail);
-      expect(isOptional()).toBe(false);
-    });
-  });
-
-  describe('disabled', () => {
```

Comment:
should we add this test back in?

Reply:
I missed that it was set in `FormField`. That's fine, then. I also forgot that this test is inside the test utils spec file so we def don't need it.